lib/ip_subnet.go

```diff
@@ -64,3 +67,48 @@ func CIDRsFromIPRangeStrRaw(rStr string) ([]string, error) {
 
 	return r.ToCIDRs(), nil
 }
+
+// CIDRToIPSubnet converts a CIDR notation to IPSubnet.
+func CIDRToIPSubnet(cidr string) (IPSubnet, error) {
```

Comment:
We should rename it to be `IPSubnetFromCidr`, that's the more common way of naming these. When we do `ToX`, it's usually by operating on a `struct` via a method instead of a function like this.